pkg/kudoctl/cmd/package_new.go

```diff
@@ -56,20 +60,45 @@ func newPackageNewCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func (pkg *packageNewCmd) validateOperatorArg(args []string) error {
+	if pkg.interactive {
+		if len(args) > 1 {
+			return errors.New("expecting at most one argument - name of the operator")
+		}
+		return nil
+	}
+
+	if len(args) != 1 {
+		return errors.New("expecting exactly one argument - name of the operator. Or use -i for interactive mode")
+	}
+	return nil
+}
+
 // run returns the errors associated with cmd env
 func (pkg *packageNewCmd) run() error {
 
 	// defaults
 	pathDefault := "operator"
 	opDefault := packages.OperatorFile{
-		Name:            pkg.name,
-		APIVersion:      reader.APIVersion,
-		OperatorVersion: "0.1.0",
-		KUDOVersion:     version.Get().GitVersion,
+		Name:              "myoperator",
+		APIVersion:        reader.APIVersion,
+		OperatorVersion:   "0.1.0",
+		AppVersion:        "0.1.0",
+		KUDOVersion:       version.Get().GitVersion,
+		KubernetesVersion: "0.16.0",
 	}
 
-	if !pkg.interactive {
+	if pkg.name != "" {
+		opDefault.Name = pkg.name
+	}
 
+	if !pkg.interactive {
+		opDefault.Maintainers = []*v1beta1.Maintainer{
+			{
+				Name:  "My Name",
+				Email: "MyEmail@invalid",
```

Comment:
is it good idea to have a default with invalid email address? 🤔 should we at least go for something valid but nonexistent (probably?)

Reply:
Well - The intention is to be invalid. It does pass the validator for the prompt, but I think it shouldn't be a "valid" email

Reply:
But if you have a better default, i'm up for that